test/testFortran.f90

```diff
@@ -139,6 +139,28 @@ program main
     call assert((xArray(i) .eq. i+value1), "Assert MeshVel u-component Velocity Fail")
     call assert((yArray(i) .eq. value2-i), "Assert MeshVel v-component Velocity Fail")
   end do 
+  
+  !VtxStrainRate needs 6 arrays, using x,y,zArrays for all 6
```

Comment:
Only in this file, change 2d array `Mesharray` name to `Mesharray_vel` and use a new 2d array `Mesharray_strainrate(numCompsStrainRate,nverts)` for "VtxStrainRate" testing with 6 values per vertex

Reply:
Changes have been made.